scripts/restore-peqp-to-static.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source .env
+NGINX_CONTAINER=$(docker ps --format '{{ .Names }}' | grep nginx)
+docker cp $1/. $NGINX_CONTAINER:/srv/static
```

Comment:
I have a slight preference against having scripts to do very simple things, like run `docker cp`. Perhaps this belongs to the runbook instead?

Reply:
Hmm, We could put this in the runbook, this script just felt natural beside the mysql backup restore script. I think we should keep it.